src/test/java/meeju/ParserTest.java

```diff
@@ -0,0 +1,43 @@
+package meeju;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+public class ParserTest {
+    private TaskList taskList = new TaskList(new StorageStub());
+    private Parser parser = new Parser();
+
+
+    //CHECKSTYLE.OFF: MethodName
+    @Test
+    public void bye_Success() {
+        assertEquals(" Bye. Hope to see you again soon!", parser.parse(taskList, "bye"));
+    }
+    @Test
+    public void incorrectCommand_Success() {
+        assertEquals("I'm sorry, I did not understand that =^..^=",
+                parser.parse(taskList, "this is an invalid command!"));
+    }
+
+    @Test
+    public void add_todo_task_Success() {
```

Comment:
I noticed that you tried to follow the method naming convention of tests, i.e. `featureUnderTest_testScenario_expectedBehavior()`.
Could each part be clearer to explain the test?

- for `featureUnderTest`, could it be `addToDo`? Having an additional `_` might result in confusion when reading the test case
- for `expectedBehavior`, would it be possible to define the expected behaviour such as `taskAdded`?